### ISSUE TITLE
Plex Media Server not scrobbling correctly when watching with iOS app

### ISSUE THREAD

**Issue by jcharr1** [Atelier-Shiori/hachidori#21]
I have Plex Media Server v1.13.5.5291 running on a Mac Mini running High Sierra 10.13.6. Today I installed Hachidori to scrobble to my AniList account. If I watch an episode on my Macbook Pro using the Plex Media Player app, it scrobbles just fine. However, if I try to watch using the Plex app on my iPhone the Correct Title popup shows up on the Mini with the episode name filled out in the search box instead of the title of the show.

**Comment by moyitpro** [comment]
While I don't have Plex Pass, I was able to test it with the iOS app and it is working properly. Make sure you are not using a Plex Agent with your media. If media streaming requires transcoding, detection will not work properly as there is no access to the source media's filename. The only thing we can base the title and episode number off of is the title of the media.

**Comment by moyitpro** [comment]
Note, there is a utility that you can use to see what the detection framework (which is used by Hachidori) is detecting. It will be helpful to see what the program actually detected.
[DetectionKit Sample.app.zip](https://github.com/Atelier-Shiori/hachidori/files/2279282/DetectionKit.Sample.app.zip)

**Comment by jcharr1** [comment]
> If media streaming requires transcoding, detection will not work properly as there is no access to the source media's filename.

It turns out this was my problem. The vast majority of my anime collection are .mkvs, and so they were being transcoded when I played them on my iOS devices. I tried reencoding one to mp4 with Handbrake, and after I renamed the original mkv file so Plex wouldn't see it Hachidori was able to match it properly. Though I think I found a whole separate issue while doing this. If I didn't rename the original file and tried to play the reencoded file by using the Play Version option in Plex, then Hachidori would crash when it tried to scrobble. 

Unfortunately, reencoding my Plex library isn't really an option. So I ended up moving to Trakt.tv and their Plex plugin which so far is scrobbling correctly no matter which device I play from. Thank you for your quick reply and trying to help, though. :)
